pkg/kudoctl/packages/resolver/resolver.go

```diff
@@ -42,8 +44,16 @@ func New(repo *repo.Client) *PackageResolver {
 func (m *PackageResolver) Resolve(name string, appVersion string, operatorVersion string) (p *packages.Package, err error) {
 
 	// Local files/folder have priority
-	if _, err = m.local.fs.Stat(name); err == nil {
-		clog.V(2).Printf("local operator discovered: %v", name)
+	_, err = m.local.fs.Stat(name)
+	// force local operators usage to be either absolute or express a relative path
+	// or put another way, a name can NOT be mistaken to be the name of a local folder
+	if filepath.Base(name) != name && err == nil {
```

Comment:
Please update the comment to this method and the help for the `cmd/install.go` command (it's partially invalid now)

Reply:
This comment is still valid.   The human operator must provide an absolute path or express a rel path (and not just the folder name)

Reply:
I meant the fact that the relative path **must** begin with `./` e.g. `./kafka` and not just `kafka` as was possible previously. In a way, this change might be even breaking for someone.

Reply:
It is not clear what you are asking for...
This could be considered a breaking change... but it is a breaking change for an operator dev...  What is the likelihood that a regular user relies not on our repo but on a local file folder... and requiring a rel or abs path is an issue?
what we do have is a number of people and an issue that explicit states that the opposite is true... people are frustrated that an operator with the same name of an existing sub folder fails.